view/frontend/templates/checkout/success.phtml

```diff
@@ -115,4 +113,3 @@
 </script>
 <div id='donation-container'></div>
 <?php endif; ?>
-
```

Comment:
@candemiralp I think the convention is to have an empty line at the end of the file?